### DISCUSSION TITLE
Mapping inaccessible locations by clicking on the map

### DISCUSSION THREAD

**Discussion by arky** [digidem/mapeo-mobile#834] [Ideas]
Sometime it is not easy to get close to a location to make an oberservation without considerable risk.  For instance, it could be a illegal land concession or logging area. In urban areas, you might be trying make observations of the rooftop solar deployments.    

Suggested Workflow: 
Select a location by clicking on the map or you can readjust the location by selecting on the map location marker on the map and moving it to adjust.

**Comment by okdistribute** [comment]
We have been talking internally about having this feature be implemented on Desktop. Would it still be usable for your partners if it was only on Desktop, or is mobile a necessity for this feature?

**Comment by arky** [comment]
@okdistribute I think it would be ideal if this feature is available on mapeo-mobile. This allows most people to make most accurate observations.

**Comment by gmaclennan** [comment]
Thanks @arky. In an earlier version of Mapeo we had an option for recording a point with a long-press on the map, but this was hard to do accurately, was not very discoverable, and was easy to trigger by accident.

We've been discussing associating two locations with an observation:

1. The location of the user
2. The location of the thing you are observing

And having a separate step in the "create observation" workflow where the user can select the nearby place they are observing, or use a map to indicate the location.

One challenge we discussed with [DevelopmentSeed](https://developmentseed.org) when we were researching using Mapeo for disaster response on World Bank projects was the difficulty for users to accurately locate things on a map, e.g. marking a plot on the map in the wrong direction. One solution we have considered is a "viewfinder" option where we use a camera view and the user points the camera at what they are mapping and selects an approximate distance, and we use the phone compass combined with the distance and the GPS location to roughly plot what they are observing on the map.
